index.js

```diff
@@ -262,8 +271,6 @@ exports.compileEVM = function (evmCode, stackTrace) {
           i = evmCode.length
         }
         break
-      case 'INVALID':
```

Comment:
possible, I also wanted to prune dead code. And if a segment as an invalid
op, then we can replace that entire segment with a single `(unreachable)`.
But it will add more complexity

On Sun, Aug 28, 2016 at 3:36 PM, Alex Beregszaszi <notifications@github.com>
wrote:

> In index.js
> <https://github.com/ewasm/evm2wasm/pull/92#discussion_r76537776>:
>
> > @@ -262,8 +271,6 @@ exports.compileEVM = function (evmCode, stackTrace) {
> >            i = evmCode.length
> >          }
> >          break
> > -      case 'INVALID':
>
> Wouldn't a simple (unreachable) be sufficient?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/ewasm/evm2wasm/pull/92/files/a9b2c4b78311bfe94a09b8062f5eb371b8b3742a#r76537776>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAJqAxn6m4mGMmMlTjO4Z7GCNnwlLeNSks5qkeNEgaJpZM4JuztA>
> .
>